src/couch_httpd_csrf.erl

```diff
@@ -63,6 +63,16 @@ validate(Csrf) when is_list(Csrf) ->
             end
     end.
 
+throw_if_mandatory(#httpd{path_parts = []}) ->
+    ok; %% Welcome message is public / entrypoint
```

Comment:
Here could be not welcome message, but database endpoint, thanks to vhosts and rewrite feature.